docs/paper/paper.md

```diff
@@ -23,38 +23,34 @@ link-bibliography: false
-`plasticparcels` has been designed as a layer over the `parcels` Lagrangian framework [@Lange2017; @Delandmeter2019]. The core functionality of `parcels` are its `FieldSet`, `ParticleSet`, and `Kernel` objects. These objects are designed to be as flexible and customisable as possible so that users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles [@Lange2017]. However, due to the flexible nature of the software, there is a steep learning curve for new users, who often find it difficult to setup their simulations in a rapid fashion due to the complexity of modern hydrodynamic model output. We have developed `plasticparcels` as user-friendly tool specifically designed for easy-to-generate plastic dispersal simulations. While `plasticparcels` is primarily designed for use in the cloud and in HPC environments (due to the typically terabyte-size of hydrodynamic datasets generated from ocean general circulation models), it can be easily installed and run on local machines. A schematic of `plasticparcels` is shown in Fig. \ref{fig:schematic}.
+Here, we have developed `plasticparcels` as an extension of the `parcels` Lagrangian framework [@Lange2017; @Delandmeter2019] in order to unify plastic dispersion modelling into one easy-to-use code. While `plasticparcels` has been designed for researchers who routinely perform plastic particle dispersion simulations, it is equally useful to novice users who are new to Lagrangian ocean analysis techniques. The core functionality of `parcels` are its `FieldSet`, `ParticleSet`, and `Kernel` objects. In the context of `plasticparcels`, the `FieldSet` contains a collection of hydrodynamic, physical, and biogeochemical fields required for plastic particle advection. The `ParticleSet` contains a set of plastic particles with their time-evolving properties, such as their position, accumulated biofilm amount, and density. A `Kernel` defines the specific computational behaviour for simulating a physical process applied to a plastic particle. These objects are designed to be as flexible and customisable as possible so that users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles [@Lange2017]. However, due to the flexible nature of the software, there is a steep learning curve for new users, who often find it difficult to setup their simulations in a rapid fashion due to the complexity of modern hydrodynamic model output. We have developed `plasticparcels` as user-friendly tool specifically designed for easy-to-generate plastic dispersal simulations. While `plasticparcels` is primarily designed for use in the cloud and in HPC environments (due to the typically terabyte-size of hydrodynamic datasets generated from ocean general circulation models), it can be easily installed and run on local machines. A schematic of `plasticparcels` is shown in Fig. \ref{fig:schematic}.
```

Comment:
Add that Kernels can do anything in parcels (not in the context of plasticparcels). And because it then becomes a very long paragraph, consider splitting?
```suggestion
Here, we have developed `plasticparcels` as an extension of the `parcels` Lagrangian framework [@Lange2017; @Delandmeter2019] in order to unify plastic dispersion modelling into one easy-to-use code. While `plasticparcels` has been designed for researchers who routinely perform plastic particle dispersion simulations, it is equally useful to novice users who are new to Lagrangian ocean analysis techniques. The core functionality of `parcels` are its `FieldSet`, `ParticleSet`, and `Kernel` objects. In the context of `plasticparcels`, the `FieldSet` contains a collection of hydrodynamic, physical, and biogeochemical fields required for plastic particle advection. The `ParticleSet` contains a set of plastic particles with their time-evolving properties, such as their position, accumulated biofilm amount, and density. A `Kernel` defines the specific computational behaviour for simulating a physical process applied to a plastic particle. 

In `parcels`, these `Kernel` objects are designed to be as flexible and customisable as possible so that users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles [@Lange2017]. However, due to the flexible nature of the software, there is a steep learning curve for new users, who often find it difficult to setup their simulations in a rapid fashion due to the complexity of modern hydrodynamic model output. We have developed `plasticparcels` as user-friendly tool specifically designed for easy-to-generate plastic dispersal simulations. While `plasticparcels` is primarily designed for use in the cloud and in HPC environments (due to the typically terabyte-size of hydrodynamic datasets generated from ocean general circulation models), it can be easily installed and run on local machines. A schematic of `plasticparcels` is shown in Fig. \ref{fig:schematic}.
```